src/Payments/CardPayment.res

```diff
@@ -180,12 +186,13 @@ let make = (
           setCvcError(_ => localeString.cvcNumberEmptyText)
           setUserError(localeString.enterFieldsText)
         }
+        if isCardSupported->Option.getOr(false)->not {
```

Comment:
We shouldn't throw this error by default. If we do not have the value of isCardSupported, then it should be true

Reply:
Resolved